src/sections/dashboard/index.js

```diff
@@ -11,207 +14,170 @@ import {
     UsersIcon,
     LikeIcon,
     ErrorIcon,
-    PersonIcon,
 } from '../../components/icons';
-import Message from './message';
-import NewUser from './newUser';
 import Tile from './tile';
+import LastMessagesList from './lastMessagesList';
+import NewUsersList from './newUsersList';
 import PageHeader from '../../components/PageHeader';
+import { fetchDashboardData } from '../../redux/actions/Dashboard';
 import './styles.css';
 
-export default function Dashboard() {
-    return (
-        <div className="react-rainbow-admin-dashboard_view-port rainbow-background-color_gray-1">
-            <PageHeader
-                title="Dashboard"
-                description="Lorem ipsum dolor sit amet, consectetur adipiscing elit." />
-
-            <section className="react-rainbow-admin-dashboard_section rainbow-align-content_space-between rainbow-p-top_large">
-                <Tile
-                    label="Orders"
-                    total="1003"
-                    icon={<ShoppingCartIcon />} />
-
-                <Tile
-                    label="Users"
-                    total="10011"
-                    icon={<UsersIcon className="react-rainbow-admin-dashboard_tile-icon" />} />
-
-                <Tile
-                    label="Followers"
-                    total="+50K"
-                    icon={<LikeIcon />} />
-
-                <Tile
-                    label="Errors"
-                    total="1"
-                    icon={<ErrorIcon />} />
-            </section>
-
-            <section className="react-rainbow-admin-dashboard_section rainbow-align-content_space-between rainbow-p-top_large">
-                <Card className="react-rainbow-admin-dashboard_card rainbow-p-vertical_medium rainbow-p-horizontal_small">
-                    <div className="rainbow-flex rainbow-justify_spread">
-                        <h1 className="react-rainbow-admin-dashboard_card--content rainbow-color_dark-1 rainbow-p-bottom_medium">$20058</h1>
-                        <Badge className="react-rainbow-admin-dashboard_badge--success">
-                            <FontAwesomeIcon icon={faArrowUp} pull="left" />
+export class Dashboard extends Component {
```

Comment:
there is not need for export here since at bottom is export default through connect hoc